Source/WebGPU/WebGPU/Pipeline.h

```diff
@@ -31,16 +31,19 @@ namespace WebGPU {
 class BindGroup;
 class ShaderModule;
 
+using BufferBindingSizesForBindGroup = HashMap<uint32_t, uint64_t, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>>;
+using BufferBindingSizesForPipeline = HashMap<uint32_t, BufferBindingSizesForBindGroup, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>>;
```

Comment:
A general (slightly nit-picky) comment: I think all these types and variables should say "MinBufferBindingSize...", since it's not the actual size of the buffer